README.md

```diff
@@ -77,7 +77,7 @@ First-time users will be particularly interested in the following guides:
 * [User Guide][user-guide]: All there is to know to put PrestaShop to good use.
 * [Updating Guide][updating-guide]: Switching to the newest version is not trivial. Make sure you do it right.
 * [Merchant's Guide][merchant-guide]: Tips and tricks for first-time online sellers.
-* The [FAQ][faq-17] and the [Troubleshooting][troubleshooting] pages should also be of tremendous help to you.
+* The [Troubleshooting][troubleshooting] page should also be of tremendous help to you.
```

Comment:
This FAQ is 5 years old. It is now outdated.

Reply:
But we have a FAQ in devdocs ?